ncm-ntpd/src/main/perl/ntpd.pm

```diff
@@ -264,6 +264,12 @@ sub write_ntpd_config
         print $fh "restrict " . $$client[0] . " mask " . $$client[1] . " nomodify notrap\n";
     }
 
+    if ($cfg->{interface}) {
+        foreach my $rule (@{$cfg->{interface}}) {
+            print $fh "interface " . $rule->{action} . " " . $rule->{match} . "\n";
```

Comment:
put the whle thing in one `"..."`, see eg lines below

Reply:
```suggestion
            print $fh "interface $rule->{action} $rule->{match}\n";
```